T/TypedSyntax/Compat.toml

```
@@ -1,5 +1,5 @@
 ["1-1.1"]
```

I think this should be
```suggestion
[1]
```
and you remove the last section?